app/views/users/show.html.erb

```diff
@@ -1,12 +1,47 @@
-<h1><%= @user.name %></h1>
-<img src="<%= @user.photo %>" alt="<%= @user.name %>">
-<p><%= @user.bio %></p>
-
-<h2>Posts by <%= @user.name %></h2>
-<ul>
-  <% @user.posts.each do |post| %>
-    <li>
-      <%= link_to "#{post.title} #{post.text}", user_post_path(@user, post) %>
-    </li>
-  <% end %>
-</ul>
+<%= stylesheet_link_tag 'user_show' %>
+
+<div class="user-show-container">
+  <div class="user-image-container">
+    <img src="<%= @user.photo %>" alt="<%= @user.name %>">
+  </div>
+
+  <div class="user-details-container">
+    <h1><%= @user.name %></h1>
+    <p class="post-count">Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
+
+<div class="user-bio-container">
+  <h2 class="h2-bio">Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class="user-posts-container">
+  <h2>
+    <% post_count = @user.posts.count %>
+    <% if post_count.zero? %>
+      No Posts
+    <% elsif post_count == 1 %>
+      Post #1
+    <% else %>
+      <%= "Post ##{post_count}" %>
+    <% end %>
+  </h2>
```

Comment:
- [optional] Hi @Xghayor, I don't think we required this logic on our view. In the previous milestone(processing data in models), we got a method that counts the number of posts a user has created. At this point, we are just supposed to utilized those methods instead of crafting them here again :+1: